appeals/web/src/server/appeals/appeal-details/appeal-details.router.js

```diff
@@ -42,7 +43,8 @@ router
 			permissionNames.viewAssignedCaseDetails
 		),
 		asyncHandler(controller.viewAppealDetails)
-	);
+	)
+	.post(asyncHandler(postCaseNote));
```

Comment:
Also as Fabio mentioned, this should have validation middleware (not sure if there are more validation cases than just making sure the comment being submitted isn't empty/whitespace)